### PR TITLE
Release 1.21.0

### PR DESCRIPTION
**Pull request by VMRuiz**

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -1,6 +1,17 @@
 Release notes
 =============
 
+1.21.0 (2024-04-18)
+
+- bug: Fix monitor ailing when operating offset-naive and offset-aware datestimes (`PR#439 <https://github.com/scrapinghub/spidermon/pull/439>`_)
```
Comment (Gallaecio) [inline CHANGES.rst:6]:
```suggestion
- bug: Fix monitor failing when operating offset-naive and offset-aware datestimes (`PR#439 <https://github.com/scrapinghub/spidermon/pull/439>`_)
```

File: CHANGES.rst [hunk 2]
```diff
@@ -1,6 +1,17 @@
...
+- bug: Fix ``calculate_field_coverage`` incorrect handling of field names ending with _items (`PR#437 <https://github.com/scrapinghub/spidermon/pull/437>`_)
+- bug: Fix TypeError in ``PeriodicItemCountMonitor`` when item_scraped_count was not defined the first time it was checked (`PR#436 <https://github.com/scrapinghub/spidermon/pull/436>`_)
+- feature: Add ``SPIDERMON_DICT_FIELDS_COVERAGE_LEVELS`` setting to limit nested dict stats computation (`PR#433 <https://github.com/scrapinghub/spidermon/pull/433>`_)
+- bug: Fix tests for minimum properties and items in Jsonschema validations (`PR#432 <https://github.com/scrapinghub/spidermon/pull/432>`_)
+- feature: Change format of content of _validation from defaultdict to dict (`PR#431 <https://github.com/scrapinghub/spidermon/pull/431>`_)  
+- bug: Fix CI/CD pipelines not working due to changes on Scrapy (`PR#426 <https://github.com/scrapinghub/spidermon/pull/426>`_)
+- feature: Add ``SendSNSNotification`` to Spidermon Notifications (`PR#413 <https://github.com/scrapinghub/spidermon/pull/413>`_)
```
Comment (Gallaecio) [inline CHANGES.rst:13]: Maybe group features first and bugs second, instead of intertwining them?

Comment (VMRuiz) [reply CHANGES.rst:13]: They are sorted by PR number as in previous releases, but I also think grouping them would make it easier to read

File: CHANGES.rst [hunk 3]
```diff
@@ -1,6 +1,17 @@
 Release notes
 =============
 
+1.21.0 (2024-04-18)
+
+- bug: Fix CI/CD pipelines not working due to changes on Scrapy (`PR#426 <https://github.com/scrapinghub/spidermon/pull/426>`_)
+- bug: Fix TypeError in ``PeriodicItemCountMonitor`` when item_scraped_count was not defined the first time it was checked (`PR#436 <https://github.com/scrapinghub/spidermon/pull/436>`_)
+- bug: Fix ``calculate_field_coverage`` incorrect handling of field names ending with _items (`PR#437 <https://github.com/scrapinghub/spidermon/pull/437>`_)
+- bug: Fix monitor failing when operating offset-naive and offset-aware datetimes (`PR#439 <https://github.com/scrapinghub/spidermon/pull/439>`_)
+- bug: Fix tests for minimum properties and items in Jsonschema validations (`PR#432 <https://github.com/scrapinghub/spidermon/pull/432>`_)
```
Comment (Gallaecio) [inline CHANGES.rst:10]:
```suggestion
- bug: Fix tests for minimum properties and items in JSON Schema validations (`PR#432 <https://github.com/scrapinghub/spidermon/pull/432>`_)
```

File: CHANGES.rst [hunk 4]
```diff
@@ -1,6 +1,17 @@
...
+- feature: Add ``SPIDERMON_DICT_FIELDS_COVERAGE_LEVELS`` setting to limit nested dict stats computation (`PR#433 <https://github.com/scrapinghub/spidermon/pull/433>`_)
```
Comment (Gallaecio) [inline CHANGES.rst:11]:
```suggestion
- feature: Add the ``SPIDERMON_DICT_FIELDS_COVERAGE_LEVELS`` setting to limit nested dict stats computation (`PR#433 <https://github.com/scrapinghub/spidermon/pull/433>`_)
```

File: CHANGES.rst [hunk 5]
```diff
@@ -1,6 +1,17 @@
...
+- feature: Add ``SendSNSNotification`` to Spidermon Notifications (`PR#413 <https://github.com/scrapinghub/spidermon/pull/413>`_)
+- feature: Change format of content of _validation from defaultdict to dict (`PR#431 <https://github.com/scrapinghub/spidermon/pull/431>`_)
```
Comment (Gallaecio) [inline CHANGES.rst:13]:
```suggestion
- feature: Change the format of the content of _validation from defaultdict to dict (`PR#431 <https://github.com/scrapinghub/spidermon/pull/431>`_)  
```

**Review by Gallaecio — APPROVED** [review]

**Pull request merged**